lago/cmd.py

```diff
@@ -398,6 +400,44 @@ def do_console(prefix, host, **kwargs):
 
 
 @lago.plugins.cli.cli_plugin(
+    help='Create Ansible hosts file of the environment'
+)
+@in_lago_prefix
+@with_logging
+def do_generate_ansible_hosts(prefix, out_format, **kwargs):
+    """
+    This method iterate through all the VMs and create an ansible
+    hosts configuration. It defines an IP address and private key
+    for the machine in group named by its type, it is printed to
+    standard output. The content looks for example like this:
+
+    [ovirt-host]
+    1.2.3.4 ansible_host=1.2.3.4 ansible_ssh_private_key_file=/path/id_rsa
+    1.2.3.6 ansible_host=1.2.3.6 ansible_ssh_private_key_file=/path/id_rsa
+    [ovirt-engine]
+    1.2.3.5 ansible_host=1.2.3.5 ansible_ssh_private_key_file=/path/id_rsa
+    """
```

Comment:
Ah, I thought it's take from the help='XXXX', not from the docstring.

The reason I've choosen the type of the vm, is for tasks such as:

```yaml
- name: Install cockpit on hosts
  yum:
    state: absent
    name: cockpit-ovirt-dashboard
  delegate_to: ovirt-host
```

I need some common group to same VMs.